client/lib/user/user.js

```diff
@@ -122,10 +121,7 @@ User.prototype.fetch = function () {
 	debug( 'Getting user from api' );
 	this.fetching = wpcom
 		.me()
-		.get( {
-			meta: 'flags',
```

Comment:
The `meta: 'flags'` param needs to stay. Otherwise the response wouldn't include the `meta.data.flags` field and anything that uses the `currentUserHasFlag` selector would break.

Reply:
Good call, restored in 45a754fdee6b5f935552459e1fd1e937587aead8